ethcore/light/src/net/mod.rs

```diff
@@ -766,15 +820,20 @@ impl LightProtocol {
 			awaiting_acknowledge: None,
 		}));
 
-		for handler in &self.handlers {
+		let any_kept = self.handlers.iter().map(
+			|handler|
```

Comment:
RFC style is to have it start on the same line as the arguments if it's a single expression, I'll change it to that. I'm OK with indenting it if it's got braces but there's no way that I can set up my editor to indent it after arguments without braces so I have to manually reindent any time I change it.